tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/BlockingRpcClient.java

```diff
@@ -188,10 +187,12 @@ protected void channelRead0(ChannelHandlerContext ctx, RpcResponse rpcResponse)
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
         throws Exception {
+      /* Current requests will be failed */
       for(ProtoCallFuture callback: requests.values()) {
         callback.setFailed(cause.getMessage(), cause);
       }
-      
+      requests.clear();
```

Comment:
I missed the above line, which marks all requests as 'failed'. This code also seem to be based on the assumption that exceptionCaught does not handle any recoverable exception. If it is right, please go ahead. Otherwise, we can fix the bug of TAJO-1569 right now, and we can handle this problem in another jira.